### DISCUSSION TITLE
One-choice models

### DISCUSSION THREAD

**Discussion by Garrid0** [lnccbrown/HSSM#603] [General]
Hello there!

Are there any plans to implement models for one-choice reaction-time tasks, such as the Shifted-Wald model?

Thank you in advance!

**Comment by AlexanderFengler** [comment]
Hi @Garrid0, 

yes this is absolutely in the plans.

Two things need to happen:

1. Slight refactor of the HSSM class to make sure nothing is currently unduly specialized to `n_choices >= 2`.
2. Implementation of the respective likelihoods.

We will prioritize 1., and are specifically hoping for user contributions for 2.. To help facilitate that, we will work on a few more tutorials that illustrate how to contribute on this front.

**Reply by Garrid0** [reply]
Thanks for your answer @AlexanderFengler! 

I have implemented the model using PyMC (using the Wald distribution). However, I think it would be much better to have it in HSSM. Indeed, I think a tutorial about how to contribute to HSSM would be great, I would be happy to help.

Keep up the amazing work!
